docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -962,13 +956,90 @@ app.get('/escrows', async (req, res) => {
 });
 ```
 
+### Deployment {#deployment}
+
+Now that you have an indexer and an API service, you can deploy your move package and start the indexer and API service.
+
+1. Install dependencies by running
+
+```
+pnpm install --ignore-workspace
+```
+
+2. Setup the database by running
```

Comment:
I missed this list the first time around. I mention below why to use `1`s on these lists and that you have to indent subparagraphs for them to render correctly.
```suggestion
1. Setup the database by running
```

Reply:
I'm having issues with this due to the reorganizing I did. Related to the slack message about nested instructions